### PR TITLE
Use Mix.install instead of packaging ez archives

### PR DESCRIPTION
**Pull request by lukaszsamson**
Implements proposal from https://github.com/elixir-lsp/elixir-ls/issues/115#issuecomment-1345409724

### CODE REVIEW

**Comment by scohen** [comment]
Oh, this is cool!

File: elixir-ls-release/debugger.exs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Mix.install([
+#   {:elixir_ls, github: "elixir-lsp/elixir-ls", ref: "7f7ba8ac7ad518945f7622369d13c16f48568b30"}
+# ])
```
Comment (scohen) [inline elixir-ls-release/debugger.exs:3]: remove?

File: elixir-ls-release/debugger.exs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  lockfile: "#{__DIR__}/../elixir-ls/mix.lock"
+)
+
+# Mix.install(
+#   [
+#     {:elixir_ls_debugger, path: "#{__DIR__}/../elixir-ls/apps/elixir_ls_debugger"},
+#   ],
+#   config_path: "#{__DIR__}/../elixir-ls/config/config.exs",
+#   lockfile: "#{__DIR__}/../elixir-ls/mix.lock"
+# )
```
Comment (scohen) [inline elixir-ls-release/debugger.exs:19]: remove?

File: elixir-ls-release/quiet_install.exs [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Mix.install([
+#   {:elixir_ls, github: "elixir-lsp/elixir-ls", ref: "7f7ba8ac7ad518945f7622369d13c16f48568b30"}
+# ])
```
Comment (scohen) [inline elixir-ls-release/quiet_install.exs:3]: remove?

File: elixir-ls-release/quiet_install.exs [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  ],
+  config_path: "#{__DIR__}/../elixir-ls/config/config.exs",
+  lockfile: "#{__DIR__}/../elixir-ls/mix.lock"
+)
+
+# Mix.install(
+#   [
+#     {:language_server, path: "#{__DIR__}/../elixir-ls/apps/language_server"},
+#   ],
+#   config_path: "#{__DIR__}/../elixir-ls/config/config.exs",
```
Comment (scohen) [inline elixir-ls-release/quiet_install.exs:17]: remove?

**Review by scohen — APPROVED** [review]
I'm really out of my depth when it comes to shell scripts, so I don't think I can give much more than a cursory "looks good to me"

File: elixir-ls-release/debugger.exs (hunk 1 above)
Comment (lukaszsamson) [reply elixir-ls-release/debugger.exs:3]: I will remove/clean this up. I need to find a way to easily switch between running local and published version

**Pull request merged**